numpy/lib/tests/test_function_base.py

```diff
@@ -601,6 +601,26 @@ def test_second_order_accurate(self):
         num_error = np.abs((np.gradient(y, dx, edge_order=2) / analytical) - 1)
         assert_(np.all(num_error < 0.03) == True)
 
+    def test_specific_axes(self):
```

Comment:
Would like to have a test checking that `axis=None` does the same as `axis` not being provided.